Library/Homebrew/extend/os/linux/extend/pathname.rb

```diff
@@ -0,0 +1,12 @@
+class Pathname
+  # @private
+  def elf?
+    read(4) == "\x7fELF"
+  end
+
+  # @private
+  def dynamic?
```

Comment:
Is this for a dynamic library? If so, would be good to be maybe `dynamic_library?` or similar?

Reply:
Both executables and libraries may be dynamic. I'd suggest `dynamic_binary` or `dynamic_elf`?

Reply:
Either of the latter two sound good to me.

Reply:
I suggest `dynamic_elf?`.

Reply:
`keebler_elf?`

Reply:
😂 You win! Make it so.